src/templates/ControlTemplate.test.jsx

```diff
@@ -64,63 +74,15 @@ test("renders each section on the page", () => {
   const privateNarrativesAccordion = within(accordionSection).getByText(
     "Private (System-Specific) Narratives"
   );
+  const expectedSubtitle = `System Control: AC-1 ${controlData.title}`;
+  const buttonElement = screen.getByText("Save & next");
   expect(implementationAccordion).toBeInTheDocument();
   expect(guidanceAccordion).toBeInTheDocument();
   expect(inheritedNarrativesAccordion).toBeInTheDocument();
   expect(privateNarrativesAccordion).toBeInTheDocument();
-});
-
-test("subtitle displays with capitalized control id and control title", () => {
```

Comment:
Hey why are some of these tests deleted out of the control template?

Reply:
Gotcha I see that you combined this into the previous test, which is fine- I initially split it apart so it was really clear what the test was checking for since there is logic to specifically format this inside the template.

Reply:
But there are a couple other tests that are deleted out of the template for checking the save and next button. Why are those removed?

Reply:
The save and next button is not a link, so testing the url it points to is not a valid test. This functionality should be controlled based on the onClickNext function, which will be completed as part of https://jiraent.cms.gov/browse/ISPGBSS-1019

Reply:
So there's functionality here that has been implemented that we need to test. If we ignore the saving part and set that to be tested in the story you linked, we at least should still be testing the rest of the functionality with the next button.

Reply:
We can mock the api call and check to make sure it has been called. We can also check to make sure that the navigate is going to the "next place"- I think there might be something involved in setting up the history for browser router?